asset-manager.php

```diff
@@ -26,13 +26,15 @@
  */
 define( 'AM_BASE_DIR', dirname( __FILE__ ) );
 
-/**
- * Load base classes
- */
-require_once AM_BASE_DIR . '/php/class-asset-manager.php';
-require_once AM_BASE_DIR . '/php/class-asset-manager-scripts.php';
-require_once AM_BASE_DIR . '/php/class-asset-manager-styles.php';
-require_once AM_BASE_DIR . '/php/class-asset-manager-preload.php';
+if ( ! class_exists( 'Asset_Manager' ) ) :
```

Comment:
Out of curiosity, why is this check needed?

Reply:
In the weird edge case that it's loaded with composer and then somehow activated through the plugin -- it's probably not needed in hindsight